docs/notebooks/at_stackexchange.ipynb

```diff
@@ -0,0 +1,1554 @@
+{
```

Comment:
This is not a tutorial. I have been writing a tutorial in the "atmodel_tutorial.ipynb" file, using the NIPS data. I'm hoping to be able to do something interesting with the StackExchange data and include that in the tutorial, I haven't been able to make the "tag prediction" thing work.